src/Upgrader/Commands/Evaluate/Analyze/AnalyzeCommand.php

```diff
@@ -111,6 +111,8 @@ public function getViolationReport(): ?ViolationReportInterface
      */
     public function execute(ContextInterface $context): ContextInterface
     {
+        $time_start = microtime(true);
```

Comment:
Why do we need it?
Should it be a normal progress bar?

Also, it's possible to use native `time` function.